MAUI/Scheduler/day-week-views.md

```diff
@@ -44,7 +44,7 @@ this.Content = scheduler;
 {% endhighlight %}
 {% endtabs %}
 
-![visible-days-count-in-maui-scheduler-days-view](images/day-week-views/number-of-visibledays-in-maui-scheduler.png)
```

Comment:
Is this reviewed content?

Reply:
I only removed the hyphen in the alt text. 
![image](https://github.com/user-attachments/assets/c2476e57-c899-4069-a566-467a5e2b689c)